application/src/main/java/org/opentripplanner/service/vehiclerental/model/RentalVehicleFuel.java

```diff
@@ -0,0 +1,40 @@
+package org.opentripplanner.service.vehiclerental.model;
+
+import javax.annotation.Nullable;
+import org.opentripplanner.transit.model.basic.Distance;
+import org.opentripplanner.transit.model.basic.Ratio;
+
+/**
+ * Contains information about the current battery or fuel status.
+ * See the <a href="https://github.com/MobilityData/gbfs/blob/v3.0/gbfs.md#vehicle_statusjson">GBFS
+ * vehicle_status specification</a> for more details.
+ */
+public class RentalVehicleFuel {
+
+  /**
+   * Current fuel percentage, expressed from 0 to 1.
+   */
+  @Nullable
+  public final Ratio percent;
+
+  /**
+   * Current fuel percentage, expressed from 0 to 1.
+   */
+  @Nullable
+  public final Distance range;
```

Comment:
This comment is for wrong field. Also, the javadoc should be put on the getters, not on the fields. These fields should also be private.